services/s3/src/main/java/software/amazon/awssdk/services/s3/internal/crt/RequestConversionUtils.java

```diff
@@ -33,130 +42,78 @@
 /**
  * Request conversion utility method for POJO classes associated with multipart feature.
  */
-//TODO: iterate over SDK fields to get the data
 @SdkInternalApi
 public final class RequestConversionUtils {
```

Comment:
It's not just a request conversion utility. Can we find a more inclusive name?

Reply:
Renamed it to `SdkPojoConversionUtils`